dockerfiles/custom_images.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
```

Comment:
I suppose it won't be merged in future if it's not properly done at the beginning.
if ppl need a fragment of script immediately they can use a gist to get that.
But when we submit something into code repository there is a review